### ISSUE TITLE
Cucumber-style like output for reporting

### ISSUE THREAD

**Issue by sleeloy-mdsol** [TheBrainFamily/cypress-cucumber-preprocessor#88]
Can we prep-pend Scenario: Feature: and add context to the Given When Then steps  when generating the tests?   **(although there's an issue open to support adding additional context to the test: https://github.com/cypress-io/cypress/issues/1910)**  
Would like to use mochawesome report generator to at least generate html on our CI.

Be nice to have the output in the cypress report to show the steps (Given, When, Then) 
i.e. output would look like this
```
Feature: I verify I can login to TestSite with my user name and password
    Scenario: Logon to TestSite and verify Test is displayed
      - Given  I open login page
      - Then  I see "TestSite| Login" in the title
      - Then  I login to TestSite with user name and password '
      - And  I click on button
      - Then  I see 'Administration' in the title
```
Another alternative is perhaps map Feature -> describe Scenario -> describe and Steps -> it  Like: 
https://www.npmjs.com/package/mocha-cucumber, when generating tests.  Cypress will then show each step as a Test in console might not be ideal though and no need to use cy.log.

**Comment by lgandecki** [comment]
I don't see a way of doing that as long as cypress makes every it separate, discussion here:

https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/pull/44

tldr: we need this to be resolved: https://github.com/cypress-io/cypress/issues/686
